node_modules/mapbox.js/docs/_plugins/insert_token.rb

```diff
@@ -0,0 +1,8 @@
+module InsertToken
+    def insert_token(input)
+        input.sub("<script>", "<script>\nL.mapbox.accessToken = '<your access token here>';")
+             .sub("<script type='text/coffeescript'>", "<script type='text/coffeescript'>\nL.mapbox.accessToken = '<your access token here>'")
```

Comment:
Line is too long. [142/80]<br>Place the . on the previous line, together with the method call receiver.